pkg/apiserver/webhook/configmap_webhook.go

```diff
@@ -0,0 +1,94 @@
+package webhook
+
+import (
+	controllers "antrea.io/nephe/pkg/controllers/cloud"
+	"antrea.io/nephe/pkg/controllers/config"
+	"context"
+	"encoding/json"
+	"fmt"
+	"github.com/go-logr/logr"
+	admissionv1 "k8s.io/api/admission/v1"
+	corev1 "k8s.io/api/core/v1"
+	"net/http"
+	"regexp"
+	controllerclient "sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+)
+
+// ConfigMapValidator is used to validate Config Map.
+type ConfigMapValidator struct {
+	Client  controllerclient.Client
+	Log     logr.Logger
+	decoder *admission.Decoder
+	Config  config.ControllerConfig
```

Comment:
I don't think you need Config field in ConfigMapValidator

Reply:
We require this for modifying the CloudResourcePrefix name

Reply:
Got it